src/char/reader.cr

```diff
@@ -14,11 +14,31 @@ struct Char
   struct Reader
     include Enumerable(Char)
 
+    # Returns the reader's String.
     getter string
+
+    # Returns the current character.
+    # ```
+    # reader = Char::Reader.new("ab")
+    # reader.current_char #=> 'a'
+    # reader.next_char
+    # reader.current_char #=> 'b'
+    # ```
     getter current_char
+
+    #:nodoc:
```

Comment:
I think we can document this method. It's basically the amount of bytes of `current_char` when encoded in UTF-8. Might be useful for someone (the standard library uses it in a couple of places).